qiskit/circuit/library/arithmetic/adder/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""The adder circuit library."""
```

Comment:
Can you add the copyright here?

Reply:
Of course, thanks for reminding and sorry that I missed that! Will push a commit with that in a sec.

Reply:
I think everything is clear here now, so I will mark the conversation as resolved for the time being :)